src/cmr/concepts/concept.js

```diff
@@ -696,9 +711,15 @@ export default class Concept {
 
           const { [ummKey]: camelCasedValue } = camelCasedObject
 
-          // Camel case all of the keys of this object (ummKey is already camel cased)
+          // If allRevisions=true need to set the
```

Comment:
If there is a problem with the conceptId being the key for setItemValue, can you just always add the revision id? Have conditional logic based on a user parameter just makes the code harder to understand.

I also don't think `Revision1` would be a good key. the user could potentially request all revisions without specifying a concept id, and multiple collections could have the same revision id, which would result in incorrect data

Reply:
With my most recent revision, I changed this slightly to be conceptId and index. Originally, the keys were all set to conceptId, which doesn't work when you're trying to create a list of revisions with the same conceptId. By adding index, we can get those unique keys so they all show up. Agreed that Revision1 wasn't the best solution here.